boards/nucleo-c031c6/include/periph_conf.h

```diff
@@ -103,9 +103,6 @@ static const uart_conf_t uart_config[] = {
  * For Nucleo-C031C6 this information is in board manual,
  * Table 11, page 20.
  *
- * STM32C031C6 do not have internal channel for VBAT, more details provided
- * in the MCU datasheet - section 3.14, page 20.
```

Comment:
Is this no longer true?

Reply:
It is true that C0 does not have an internal channel for VBAT.  It is not true that more details are provided in the datasheet. 

At the time of the initial C011 and C031 support, this was a relevant difference from other ST parts (like the G0 family), but now it seems that it's common to all the C0 family parts.  I figured it wasn't good practice to list all the things the micro _doesn't_ have as that list would forever be incomplete.  :)